Source/Data/Remotes/ParleyRemote.swift

```diff
@@ -4,9 +4,60 @@ import ObjectMapper
 import UIKit
 import Foundation
 
+
+public typealias ParleyRemoteInterceptor = ParleyRemoteRequestInterceptorProtocol & ParleyRemoteResponseInterceptorProtocol
+
+public protocol ParleyRemoteRequestInterceptorProtocol: AnyObject {
+    func httpRequestWillStart(with request: inout URLRequest) throws
+}
+
+public protocol ParleyRemoteResponseInterceptorProtocol: AnyObject {
+    func httpResponseReceived(response: HTTPURLResponse) throws
+}
```

Comment:
We would like to suggest different names; namely:

- Remove the `Protocol` suffix of the request/response protocol name
- `ParleyRemoteRequestInterceptor.modifyOutgoing(request)`
- `ParleyRemoteResponseInterceptor.didReceive(response)`

Reply:
Looks as a good suggestion!